packages/ontario-design-system-complete-styles/gulpfile.js

```diff
@@ -175,9 +144,15 @@ task(
 		'scripts-move',
 		'favicons-move',
 		'sass:copy-dist',
-		'sass:build-minify',
+		'sass:build',
+		'sass:minify',
 		'clean:src',
 	),
 );
 
+task('watch', (done) => {
+	watch(paths.styles.dir, { ignoreInitial: false }, parallel('sass:build-minify'));
```

Comment:
the `sass:build-minify` command no longer exists, should this include both `sass:build` and `sass:minify` instead?